packages/snap/src/index.ts

```diff
@@ -5,26 +5,26 @@ import {
 import type {
   OnKeyringRequestHandler,
   OnRpcRequestHandler,
-} from '@metamask/snaps-types';
+} from '@metamask/snaps-sdk';
 
 import { SimpleKeyring } from './keyring';
 import { logger } from './logger';
 import { InternalMethod, originPermissions } from './permissions';
 import { getState } from './stateManagement';
 
-let keyring: SimpleKeyring;
+let keyringInstance: SimpleKeyring;
```

Comment:
Mainly renaming this one to allow `const keyring = getKeyring()` notation (which seems more "natural", and since this one acts as singleton pattern, `keyringInstance` makes total sense to me)